packages/terra-doc-template/examples/test-examples/DefaultDocTemplate.example.jsx

```diff
@@ -21,7 +21,13 @@ const Index = () => {
 
   return (
     <div id="DocTemplateContainer">
-      <DocTemplate packageName="test-package-name" readme={readme} propsTables={propsTables} examples={examples} />
+      <DocTemplate
+        packageName="test-package-name"
+        readme={readme}
+        repositoryURL="git+https://github.com/cerner/terra-core.git"
```

Comment:
Looking in our packages, I wonder if this needs updated to be `https://github.com/cerner/terra-core.git`